src/rgw/rgw_admin.cc

```diff
@@ -7807,6 +7808,30 @@ int main(int argc, const char **argv)
     int index = 0;
     bool truncated;
     bool processing_queue = false;
+
+    #ifdef WITH_RADOSGW_MOTR
+    ldpp_dout(dpp(), 20) << "Listing GC objects from Motr" << dendl;
+    std::vector<std::unordered_map<std::string, std::string>> gc_entries;
+    formatter->open_array_section("entries");
+    int ret = static_cast<rgw::sal::MotrStore*>(store)->list_gc_objs(dpp(), gc_entries);
```

Comment:
we can avoid an intermediate function here if we want to, 
```suggestion
    int ret = static_cast<rgw::sal::MotrStore*>(store)->get_gc()->list(gc_entries);
```

Reply:
`list_gc_objs()` ensures that flow for gc apis goes through SAL.
Its same in terms of number of function calls.